docs/conf.py

```diff
@@ -32,13 +34,13 @@
 
 # intersphinx allows referencing other packages sphinx docs
 intersphinx_mapping = {
-    "python": ("https://docs.python.org/3.9", None),
+    "python": ("https://docs.python.org/3/", None),
     "traitlets": ("https://traitlets.readthedocs.io/en/stable/", None),
-    "ctapipe": ("https://ctapipe.readthedocs.io/en/v0.20.0/", None),
+    "ctapipe": ("https://ctapipe.readthedocs.io/en/v0.24.0/", None),
 }
 
```

Comment:
To fix sphinx warnings:

```suggestion

nitpick_ignore = [
    # ProtozfitsDL0EventSource and ProtozfitsDL0TelescopeEventSource shared traits
    ('py:obj', 'atmosphere_density_profile'),
    ('py:obj', 'cross_validation_lock'),
    ('py:obj', 'is_stream'),
    ('py:obj', 'obs_ids'),
    ('py:obj', 'simulated_shower_distributions'),
    ('py:obj', 'simulation_config'),
    
    # Group traits for both EventSource classes using wildcards
    ('py:obj', 'ctapipe_io_zfits.ProtozfitsDL0EventSource.*'),
    ('py:obj', 'ctapipe_io_zfits.ProtozfitsDL0TelescopeEventSource.*'),
]        
```

Reply:
on it...